src/_includes/compass_teaser.html

```diff
@@ -1,6 +1,6 @@
 <section class="{{ include.class }}">
   <div class="compass-teaser">
-    {%- assign anchor_id = {site.translations[site.lang].software-modernisation.page-sections.compass} -%}
+    {%- assign anchor_id = all-service-lines.page-sections.compass -%}
```

Comment:
Where is `all-service-lines` defined? If we have a Compass Teaser outside of the Service Line pages will this still work/make sense?